otter/utils.py

```diff
@@ -125,9 +125,9 @@ def get_source(cell):
     """
     source = cell.source
     if isinstance(source, str):
-        return source.split('\n')
+        return re.split("\r?\n", source)
     elif isinstance(source, list):
-        return [line.strip('\n') for line in source]
+        return [re.split("\r?\n", line) for line in source]
```

Comment:
I think this would create a nested list, right? I think it would be better to use `re.sub(r"[\r\n]", "", line)` instead, which would get rid of any returns or newlines.

Reply:
Ah. That explains the problems I had during testing. I'll change it as proposed by you 👍